### DISCUSSION TITLE
How to use custom right menu then go back to initial right menu

### DISCUSSION THREAD

**Discussion by Antoine-S-C** [cwapi3d/cwapi3dpython#216] [Q&A] [answered]
Hello,
I'm trying to use the right menu with **menu_controller.display_simple_menu()**
It works, but then I don't know how to go back to the initial menu.
How can I do that ?

**Comment by ancelin-charpentier** [comment]
Hey Antoine,
Creating a button that points to nowhere called "Return" is achieving what you are looking for I guess.
```python
menu_items = ['1st button text',
              '2nd button text',
              '',
              'Return']

selected_menu_item = menu_controller.display_simple_menu(menu_items)
```

**Reply by Antoine-S-C** [reply]
I already tried this but it doesn't work.

**Comment by Brunner246** [comment]
https://docs.cadwork.com/projects/cwapi3dpython/en/latest/examples/menu_example/

**Reply by Antoine-S-C** [reply]
Hello @Brunner246
With the code of your link ("create a simple cadwork menu"), I have the same problem :
after clicking on "Return", the default right menu does not reappear.
Do you have any idea?
Shouldn't we add the "display_default_menu()" function in menu_controller library?

**Reply by Brunner246** [reply, marked as answer]
Hi @Antoine-S-C 

From where are you running your code ? Do not run the menu_controller from the Python IDLE. Run it as a plugin from `userprofil_30\3d\API.x64\yourPlugin`

![menu](https://github.com/user-attachments/assets/3d78db89-6967-43dd-9e86-78219a6908e2)

What dou you mean with _Shouldn't we add the "display_default_menu()" function in menu_controller library?_ There is no `display_default_menu`
 [menu_controller](https://docs.cadwork.com/projects/cwapi3dpython/en/latest/documentation/menu_controller/)

please post your code

**Reply by Antoine-S-C** [reply]
Ah ! Yes, that was just the reason ! I Used the python IDLE, as I do with all my tests.
Problem solved using the top menu. I'll think about it for next time.
Thanks a lot
:)
